packages/vega-parser/src/parsers/axis.js

```diff
@@ -44,6 +44,10 @@ export default function(spec, scope) {
       titlePadding: encoder(_('titlePadding')),
       minExtent:    encoder(_('minExtent')),
       maxExtent:    encoder(_('maxExtent')),
+      ariaLabel:    encoder(_('ariaLabel')),
+      ariaRole:     encoder(_('ariaRole')),
+      ariaHidden:   encoder(_('ariaHidden')),
+      tabindex:     encoder(_('tabindex')),
```

Comment:
`tabIndex`?

Reply:
Maybe. we also have `zindex` and there is already `tabindex` on the web https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex.